src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method must be divided into 3 logical methods. As a result, the getStatistic() method will contain only calls to these methods and will look something like:
String data = readFromFile(fromFileName);
String report = createReport(data);
writeToFile(toFileName, report);